src/actions/actionUtil.js

```diff
@@ -0,0 +1,96 @@
+import ActionTypes from './actionTypes';
+
+const TIMEOUT_MS = 5000;
+
+function handlePagedRequest(item, url, type, offset, limit) {
+    return dispatch => {
+        return fetch(getAllUrl(url, offset, limit), {
+            method: type
+        }).then(res => {
+            return res.json();
+        }).then(json => {
+            if (json.error || !json.success) {
+                dispatch(errorPaged(item, offset, limit, json.message));
+                return;
+            }
+
+            dispatch(handlePaged(item, json, offset, limit));
+        }).catch(err => {
+            dispatch(errorPaged(item, offset, limit, err));
+        });
+    };
+}
+function handleRequest(url, type, body) {
+    let raw = '';
+    if (body) {
+        raw = JSON.stringify(body);
+    }
+    return dispatch => {
+        setTimeout(() => {
+            dispatch(timeout());
+        }, TIMEOUT_MS);
```

Comment:
This resets the `modify` state to it's default state after a number of ms. Atm it helps remove error/success messages after they cease to be useful. In the future we could use it to determine if the server is offline or something.